local.conf.example_staging

```diff
@@ -0,0 +1,154 @@
+
+# Sample ``local.conf`` for user-configurable variables in ``stack.sh``
+
+# NOTE: Copy this file to the root DevStack directory for it to work properly.
+
+# ``local.conf`` is a user-maintained settings file that is sourced from ``stackrc``.
+# This gives it the ability to override any variables set in ``stackrc``.
+# Also, most of the settings in ``stack.sh`` are written to only be set if no
+# value has already been set; this lets ``local.conf`` effectively override the
+# default values.
+
+# This is a collection of some of the settings we have found to be useful
+# in our DevStack development environments. Additional settings are described
+# in https://docs.openstack.org/devstack/latest/configuration.html#local-conf
+# These should be considered as samples and are unsupported DevStack code.
+
+# The ``localrc`` section replaces the old ``localrc`` configuration file.
+# Note that if ``localrc`` is present it will be used in favor of this section.
+[[local|localrc]]
+
+# Minimal Contents
+# ----------------
+#IPV4_ADDRS_SAFE_TO_USE=192.168.100.0/24
+# While ``stack.sh`` is happy to run without ``localrc``, devlife is better when
+# there are a few minimal variables set:
+RABBIT_HOST=192.168.122.213
+#RABBIT_HOST=10.0.0.10
+#MYSQL_HOST=10.238.153.51
+
+# If the ``*_PASSWORD`` variables are not set here you will be prompted to enter
+# values for them by ``stack.sh``and they will be added to ``local.conf``.
+ADMIN_PASSWORD=nomoresecret
+DATABASE_PASSWORD=123
+#RABBIT_USERID=clearstack
+RABBIT_PASSWORD=123
+SERVICE_PASSWORD=123
+
+USE_PYTHON3=False
+ENABLED_PYTHON3_PACKAGES=""
+
+LIBS_FROM_GIT=python-cinderclient,python-glanceclient,python-neutronclient,python-novaclient,python-openstacksdk,neutron-lib
+
+STX_BASE=https://github.com/starlingx-staging
+
```

Comment:
Since the only real difference between these two files is this repo/branch section, just make one file and comment that enabling these lets you run staging and commenting them out lets you run upstream.